.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -21,15 +21,18 @@ body:
-        [2025-05-07T10:06:22.737Z] [INFO] [Diagnostic]: Running diagnostic tool
-        # XcodeBuildMCP Diagnostic Report
-        
+        XcodeBuildMCP Doctor
+
+        Generated: 2025-08-11T17:42:29.812Z
+        Server Version: 1.11.2
+
+        ## System Information
+        - platform: darwin
+        - release: 25.0.0
+        - arch: arm64        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)